openassessment/templates/openassessmentblock/message/oa_message_unavailable.html

```diff
@@ -1,9 +1,9 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">{% trans "Instructions Unavailable" %} </h3>
```

Comment:
This should be an h4. It's shown above the steps, like the other message variations above.